libs/database/src/lib/entities/polygon-geometry.entity.ts

```diff
@@ -22,4 +23,88 @@ export class PolygonGeometry extends Model<PolygonGeometry> {
   @AllowNull
   @Column(BIGINT.UNSIGNED)
   createdBy: number | null;
+
+  static async checkIsSimple(polygonUuid: string): Promise<boolean> {
```

Comment:
Let's move all static methods above the field definitions to be consistent with other models.